ot/stochastic.py

```diff
@@ -441,10 +444,10 @@ def solve_semi_dual_entropic(a, b, M, reg, method, numItermax=10000, lr=None,
           a[:, None] * b[None, :])
 
     if log:
-        log = {}
-        log['alpha'] = opt_alpha
-        log['beta'] = opt_beta
-        return pi, log
+        log_stoc = {}
+        log_stoc['alpha'] = opt_alpha
+        log_stoc['beta'] = opt_beta
```

Comment:
`log_stoc = {'alpha': opt_alpha, 'beta': opt_beta}` ?

Reply:
Are `alpha` and `beta` in log-domain ?

Reply:
@hichamjanati What do you mean ? They are the usual dual variables.

Reply:
Ok so in log-domain (plan given by `exp(alpha / eps) K exp (beta / eps). 
(Some Sinkhorn funcs return `exp(alpha / eps)` instead of `alpha` which should be normalized to `alpha` everywhere)

Reply:
I agree, Sinkhorn functions return `exp(alpha/eps)` instead of alpha and it should be clarified in the documentation or normalized to `alpha`. I also prefer normalizing to `alpha`.

Regarding ot.stochastic, it returns the dual potentials `alpha`, `beta` (you can see when we compute the OT plan).